index.html

```diff
@@ -12,27 +12,27 @@
 <body>
     <header>
         <nav class="header__nav">
-                <div>
-                    <div class="burger">
-                        <a class="burger__link" href="">
-                            Главное меню
-                        </a>
-                    </div>
-                <ul class="header__list">
-                    <li class="header__item"><a class="header__href"  href="">Главная</a></li>
-                    <li  class="header__item"><a class="header__href" href="">Правообладателям</a></li>
-                    <li  class="header__item"><a class="header__href" href="">Заметили ошибку? <span class="header__href_hidden">Сообщите нам!</span> </a></li>
-                    <li  class="header__item"><a class="header__href" href="">Стол заказов</a></li>
-                    <li  class="header__item"><a class="header__href" href="">Мы Вконтакте | Твиттере</a></li></ul>
+            <div>
+                <div class="burger">
+                    <a class="burger__link" href="">
+                        Главное меню
+                    </a>
                 </div>
-                    <div class="header__right">
-                        <form class="header__form" action="https://jsonplaceholder.typicode.com/posts" method="post">
-                            <input aria-label="Найти фильм" class="header__search" type="text" name="search reqest">
-                            <button class="header__btn">Поиск</button>                         
-                        </form>
-                        <button class="header__btn header__btn_less310px">Найти фильм</button>
-                    </div>
-            </nav>
+            <ul class="header__list">
+                <li class="header__item"><a class="header__href"  href="">Главная</a></li>
+                <li  class="header__item"><a class="header__href" href="">Правообладателям</a></li>
+                <li  class="header__item"><a class="header__href" href="">Заметили ошибку? <span class="header__href_hidden">Сообщите нам!</span> </a></li>
+                <li  class="header__item"><a class="header__href" href="">Стол заказов</a></li>
+                <li  class="header__item"><a class="header__href" href="">Мы Вконтакте | Твиттере</a></li></ul>
+            </div>
+                <div class="header__right">
+                    <form class="header__form" action="https://jsonplaceholder.typicode.com/posts" method="post">
```

Comment:
```suggestion
                    <form 
                        class="header__form" 
                        action="https://jsonplaceholder.typicode.com/posts" 
                        method="post"
                    >
```

Also is just suggestion in formating, it is better to put atributes one under another. It is esier to read